facade/service.go

```diff
@@ -575,6 +606,15 @@ func (f *Facade) GetServices(ctx datastore.Context, request dao.EntityRequest) (
 	}
 }
 
+// GetServices will get all the services
```

Comment:
We have a GetServices call that almost does the same thing.  Make sure to note that this call returns incomplete or "raw" services data if it has to be exposed publically.